src/server/modules/user/UserController.ts

```diff
@@ -91,14 +92,15 @@ export class UserController {
         return
       }
       if (error instanceof AlreadyExistsError) {
-        res.badRequest(jsonMessage(error.message, MessageType.ShortUrlError))
+        const newErrorMessage = `${error.message} Click here to find out more` // TODO: shift message to frontend
```

Comment:
Please help to create a Notion task instead

Reply:
removed TODO comment in [c2e319e](https://github.com/opengovsg/GoGovSG/pull/2069/commits/c2e319e3f149624b992d8f31d463426887dce30b), have created notion task